### ISSUE TITLE
Create customized dashboards 

### ISSUE THREAD

**Issue by kakopappa** [sinricpro/feature-requests#70]
add the ability to select the fav devices and make customized dashboards for the app. 

Google Home and Alexa app does this already.

**Comment by selimkabalan** [comment]
yes you are right ! it is already available in Google Home and Alexa. However, many Sinric sensors doesn't work with google home or alexa. So we rely on Sinric app and Dashboard more than we use google home.

**Comment by kakopappa** [comment]
Hi 

“ Sinric sensors doesn't work with google home or alexa”

Which sensors doesn’t work exactly?  Can you elaborate the problem please
